test/UpdatedPackagesCollector.js

```diff
@@ -1,10 +1,84 @@
+// mocked modules
+import GitUtilities from "../src/GitUtilities";
+
 // file under test
 import UpdatedPackagesCollector from "../src/UpdatedPackagesCollector";
 
+jest.mock("../src/GitUtilities");
+
+const filteredPackages = [{
+  name: 'package-1',
+  location: 'location-1'
+}, {
+  name: 'package-2',
+  location: 'location-2'
+}];
+
+const logger = {
+  silly: () => {},
+  info: () => {},
+  verbose: () => {}
+};
+
+const repository = {
+  rootPath: 'root-path'
+};
+
 describe("UpdatedPackagesCollector", () => {
   it("should exist", () => {
     expect(UpdatedPackagesCollector).toBeDefined();
   });
 
   it("needs better tests");
```

Comment:
It now has them, thank you! (you can delete this empty test case, if you want)

Reply:
Heh, I left that in because I agree with the sentiment 😉 

Will remove.